src/dark_mode_toggle.tsx

```diff
@@ -3,7 +3,7 @@ import equal from 'fast-deep-equal'
 import React, { useCallback, useEffect, useMemo, useState } from 'react'
 import { IconButton } from 'storybook/internal/components'
 import { DOCS_RENDERED, SET_STORIES, STORY_CHANGED } from 'storybook/internal/core-events'
-import { type API, useParameter } from 'storybook/manager-api'
+import { addons, type API, useParameter } from 'storybook/manager-api'
```

Comment:
The import should be sorted.
If you have `biome` extension install, it will automatically fix this.